### ISSUE TITLE
Hacktoberfest 2019

### ISSUE THREAD

**Issue by VivekKumarSingh07** [VivekKumarSingh07/Leetcode_Java#1]
This repo is open for Hacktoberfest. Happy Hacking.

**Comment by ConnectNitish** [comment]
I would like to provide some inputs/programs on the same.

**Comment by VivekKumarSingh07** [comment]
Yeah sure just fork the repo and create PR when you have done.

**Comment by ductoan2732000** [comment]
18/5000
Congratulations hacking

**Comment by codeghoul** [comment]
I would like to add my solutions/programs here.

**Comment by VivekKumarSingh07** [comment]
yeah just fork this repo make the changes and create Pull Request

**Comment by Rita-Okonkwo** [comment]
Hi Vivek. I would love to contribute to this repo. How will I know which problem to solve?

**Comment by VivekKumarSingh07** [comment]
Just look for any leetcode problem which is currently not present in this repo, Write code in accordance with the format specified and Create a pull request. Cheers.
